Makefile

```diff
@@ -30,7 +30,6 @@ services: python
 .PHONY: db
 db: args?=upgrade head
 db: python
-	@tox -qqe dev --run-command 'initdb conf/development.ini'  # See setup.py for what initdb is.
```

Comment:
I don't know if this will break any existing workflows but at the moment if trying to auto generate a migration for a new model 

- initdb will silently create the table
- alembic will see the new table already in the DB and create an empty migration.

It might be that make db it's trying to do many things? both "init the db" and "any alembic command"?

Reply:
I think `make db` needs to continue running `initdb` because the purpose of `make db` is to give team members a foolproof way to fix their DB when it might not be initialized or might not be on the latest migration version. Without it some members of the frontend team become helplessly confused when they haven't initialized their DB for some reason or when we've merged a DB migration. See this thread for example: <https://hypothes-is.slack.com/archives/CBN3DGW02/p1588329090111400?thread_ts=1588283479.110200&cid=CBN3DGW02>. Similar things had happened again and again.

But I wonder if `make db` should just hardcode the `alembic upgrade head` command and there should be a separate `make alembic` or `make migrate` for passing custom commands into alembic?

Reply:
Kept make db unchanged and added `make alembic`